testData/solutions/linux.json

```diff
@@ -377,9 +377,6 @@
         "settingsHandlers": {
             "configuration": {
                 "type": "gpii.orca",
-                "options": {
```

Comment:
No, there's no linux side for this. As I mentioned back in the meeting we talked about this thing I removed this constraint in Orca settings handler during the last update but I forgot to remove the universal "bits" of it. I agree that this can be ready to go in as a separate pull request since we still need to make some accommodations in UNIVERSAL before the real issue (GPII-2046) can be addressed. Let me know if you want me to make a new JIRA/pr and get this merged ASAP.